templates/graph-theory.md

```diff
@@ -12,10 +12,125 @@ keywords:
   - topological sort
 ---
 
+
+
+### Depth First Traversal
+
+Go to [Tutorial](../tutorials/graph-theory/binary-tree#traversal)
+
+
+#### Preorder traversal
+<Tabs>
+
+<TabItem value="cpp" label="C++">
+
+```cpp
+void preorder(TreeNode* node) {
+    if (node == NULL) return;
+    s.push_back(node->val);
+    preorder(node->left);
+    preorder(node->right);
+}
+```
+</TabItem>
```

Comment:
add a python version as well

Reply:
Did all the changes and added python version as well.